drivers/hmis/app/graphql/mutations/submit_form.rb

```diff
@@ -60,7 +60,7 @@ def resolve(input:)
       form_validations = form_processor.collect_form_validations
       errors.push(*form_validations)
 
-      # Run processor to create/update record(s)
+      # Run processor to assign attributes to the record(s)
```

Comment:
Double check me here please. Does processor.run? actually persist data?

Reply:
you're correct, it just assigns the attributes. thanks!